code/modules/clothing/neck/_neck.dm

```diff
@@ -291,3 +291,110 @@
 	desc = "A sizable white cape with gold connects."
 	icon_state = "grandadmiral"
 	item_state = "grand_admiral"
+
+/*
+ * Bodycamera stuff
+ * NOTE: Unlike regular cameras, bodycams can only stream to one camera network at a time. Unless you want to give it more networks,
+ * the code will only ever worry about bodcam.network[1]
+ */
+
+/obj/item/clothing/neck/bodycam
+	name = "body camera"
+	desc = "A wearable camera, capable of streaming a live feed."
+	icon_state = "bodycam_off"
+	item_state = "bodycam_off"
+	var/prefix = "sec"//used for sprites, miner etc
+	var/obj/machinery/camera/bodcam = null
+	var/setup = FALSE
+	var/preset = FALSE //if true, the camera is already configured and cannot be reset
+	actions_types = list(/datum/action/item_action/toggle_bodycam)
+	strip_delay = 10 //takes one second to strip, so a downed officer can be un-cammed quickly
+	w_class = WEIGHT_CLASS_BULKY //you HAVE to wear it or carry it. No cheating by putting it in a bag!
```

Comment:
(suggestion) This is a bit messy, wouldnt it be a better idea to make it always disabled when unequipped and always enabled when equipped?

Reply:
But I want it to be able to be worn but disabled. Like if you want to disable it temporarily